### DISCUSSION TITLE
Error when when launching AirSim ROS node

### DISCUSSION THREAD

**Discussion by abdelhakim96** [microsoft/AirSim#3743] [Support Q&A]
When i launch the command for the example ROS node while UE is running with the drone :

roslaunch airsim_ros_pkgs airsim_node.launch

I get the following error
terminate called after throwing an instance of 'std::invalid_argument'   what():  Unknown SimMode: , failed to set default vehicle settings Aborted (core dumped)

**Comment by Nish1no7** [comment]
May I ask if you have resolved it?

**Reply by Dawn-Flying** [reply]
The value of SimMode needs to be specified in settings.json. After changing settings.json, you need to restart the airsim simulation environment program and prompt the loaded settings.json path when starting
